libsrc/allegro/src/allegro.c

```diff
@@ -75,84 +75,14 @@ PALETTE _current_palette;
 int _current_palette_changed = 0xFFFFFFFF;
 
 
-PALETTE desktop_palette = 
+PALETTE desktop_palette =
 {
-   { 63, 63, 63, 0 },   { 63, 0,  0,  0 },   { 0,  63, 0,  0 },   { 63, 63, 0,  0 },
-   { 0,  0,  63, 0 },   { 63, 0,  63, 0 },   { 0,  63, 63, 0 },   { 16, 16, 16, 0 },
-   { 31, 31, 31, 0 },   { 63, 31, 31, 0 },   { 31, 63, 31, 0 },   { 63, 63, 31, 0 },
-   { 31, 31, 63, 0 },   { 63, 31, 63, 0 },   { 31, 63, 63, 0 },   { 0,  0,  0,  0 }
+  { 255, 255, 255, 0 },{ 255, 0,  0,  0 },{ 0,  255, 0,  0 },{ 255, 255, 0,  0 },
+  { 0,  0,  255, 0 },{ 255, 0,  255, 0 },{ 0,  255, 255, 0 },{ 65, 65, 65, 0 },
+  { 125, 125, 125, 0 },{ 255, 125, 125, 0 },{ 125, 255, 125, 0 },{ 255, 255, 125, 0 },
+  { 125, 125, 255, 0 },{ 255, 125, 255, 0 },{ 125, 255, 255, 0 },{ 0,  0,  0,  0 }
 };
```

Comment:
Apparently, `desktop_palette` may also be removed, as it's not used anywhere in our stripped allegro sources.